src/openqa_client/client.py

```diff
@@ -377,3 +377,34 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sorted_key: Callable = float,
+        default_return: str = "0",
+    ) -> str:
+        """Identify latest build number in target Job Group
+
+        Args:
+            group_id (int): Job Group ID
+            all_passed (bool, optional): Controls whether just last build will be selected
+                or the last with all jobs passed. Defaults to True.
+            sorted_key (Callable, optional): To find the latest build we need to order the builds.
+                The specified callback will be used in the sorted function to sort builds.
+                Defaults to float.
+            default_return (str, optional): In case all available builds have no acceptable format
+                or there are no builds in this job group this value specifies what to return.
+                Defaults to "0".
+
+        Returns:
+            str: string representation of last BUILD
+        """
+        resp = self.openqa_request("GET", f"/group_overview/{group_id}.json")
+        if all_passed:
+            passed = [r for r in resp["build_results"] if r["all_passed"] == 1]
+            builds = [r["build"] for r in passed if r["build"].isdigit()]
+        else:
+            builds = [r["build"] for r in resp["build_results"] if r["build"].isdigit()]
+        return list(sorted(builds, key=sorted_key))[-1] if builds else default_return
```

Comment:
`list()` seems not needed here, it's entirely redundant. just `return sorted(builds, key=sorted_key)[-1] if builds else default_return` would be fine, I think.

Reply:
done